packages/components/src/constants/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from "./pageLayouts"
+export * from "./componentTypes"
+export * from "./childrenPages"
+export * from "./infobar"
+export * from "./dynamicDataBanner"
+export * from "./contactInformation"
```

Comment:
question: having these constants exported from its own namespace feels quite weird and violates our principle of each component being isolated building blocks. I also noticed that many (if not all) of them can be derived some way or another from the schema itself.

Can these constants be eliminated, or otherwise stored within each individual component's folder?

Reply:
@dcshzj yeah true, moved it as a constants files under the respective schema and importing from them directly https://github.com/opengovsg/isomer/pull/1703/commits/3c6a087091fe180ba1c4f2ba95dc9dde73d76e67